### ISSUE TITLE
Latest tagged build is way too old

### ISSUE THREAD

**Issue by verhas** [MacPass/MacPass#1116]
- **Describe the bug**

The latest build I can download from the CI server is even older that the release version.

**To Reproduce**

1. open the page https://github.com/MacPass/MacPass/releases/tag/continuous
2. See that the latest release is older than the downloadable release 0.7.12
3. Click on TAGs
4. See that there is no newer snapshot build than the current release

**Expected behavior**
I would like to download the latest snapshot build.

**Screenshots**

![image](https://user-images.githubusercontent.com/1597979/100725244-168edc80-33c4-11eb-8e9c-806e2d228e8a.png)

**Additional context**
Maybe I am mistaken

**Comment by mstarke** [comment]
You are not mistaken. I did stop pushing those since the travis setup wasn't supporting my toolchain at the time (at least that is what I remember). I since did not find the time to re-enable the system but it should run ok with small modifications. What I was considering was moving to GitHub actions but haven't found the time to look into the matter either. The last couple of months were very slow in regards to the progress of MacPass but I've found time again and picked up development so this should get fixed as well…
